code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -389,17 +389,17 @@ Contains most of the procs that are called when a mob is attacked by something
 		update_inv_w_uniform()
 
 
-//This looks for a "marine", ie. non-civilian ID on a person. Used with the m56 Smartgun code.
+//This looks for a "marine", ie. non-civilian ID on a person. Used with the gun, claymore, sentry and possibly other IFF code.
 //Does not actually check for station jobs or access yet, cuz I'm mad lazy.
 //Updated and renamed a bit. Will probably updated properly once we have a new ID system in place, as this is just a workaround ~N.
```

Comment:
These can all be combined into an autodoc via
```
/** 
Thing I want to say here
  */

Reply:
changed